README.md

```diff
@@ -27,7 +27,7 @@ The Ruby special interest group (SIG) meets regularly. See the OpenTelemetry
 
 Approvers ([@open-telemetry/ruby-contrib-approvers](https://github.com/orgs/open-telemetry/teams/ruby-contrib-approvers)):
 
-- [Robb Kidd](https://github.com/robbkidd), Honeycomb
+- (Could _your_ name could appear here?)
```

Comment:
Felt odd to leave it empty and also odd to remove the section entirely. What's here is intended to welcome new contributors and it is certainly open to word-smithing feedback and alternatives.

Reply:
Ugh. "Could" appears twice in this sentence. What even are words?